app/src/main/java/ch/epfl/sdp/db/LeaderoardViewModel.java

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.sdp.db;
+
+import android.app.Application;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.LiveData;
+
+import java.util.List;
+
+public class LeaderoardViewModel extends AndroidViewModel {
```

Comment:
Honeslty, all the nested dependencies are a little bit hard to read (lot of new classes)

Reply:
This is how the Room(Sqlite database) is used. As suggested by Android developer website.